src/rhsm/connection.py

```diff
@@ -775,9 +775,12 @@ def _load_manager_capabilities(self):
         """
         self.capabilities = {}
         self.capabilities = self.conn.request_get("/status/")
```

Comment:
The initial set to {} seems unneeded.
Just return the results of get /status here (or just the part you need).

maybe
```
return status_results.get('managerCabalities', [])
```